shivakrishna/Assignment2/SWOT.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8"?>
+<GridLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:columnCount="2"
+    android:rowCount="4">
+
+    <TextView
+        android:id="@+id/strength"
+        android:layout_width="204dp"
+        android:layout_height="40dp"
+        android:text="STRENGTHS"
+        android:textSize="20sp"
+        android:textStyle="italic"
+        android:layout_column="0"
+        android:layout_row="0"
+        android:background="#005005"/>
+
+    <EditText
+        android:id="@+id/tS"
+        android:layout_width="204dp"
+        android:layout_height="wrap_content"
```

Comment:
Did I not mention to set height to 0 dp when using rowWeight? Run your app once and test it by giving multiple lines. You will understand what I'm trying to say. Your views should not be wrapped as text increases in them. Height and width should spread across the screen evenly for all screen sizes.
Although I dint explicitly mention about width (when I mentioned about height), it should be incurred that you need to set weight for width as well. Hardcoding width and height will result in distorted views in various screen sizes. For example, this is how your layout looks llike on Nexus 7 (or Nexus 6). If you need help ask on gitter chatroom.
![Imgur](http://i.imgur.com/nq8RBigm.png)